internal/repository/user.go

```diff
@@ -0,0 +1,122 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/dilyara4949/flight-booking-api/internal/domain"
+	"github.com/google/uuid"
+)
+
+type userRepository struct {
+	db             *sql.DB
+	contextTimeout time.Duration
```

Comment:
why do you store ctx timeout here?

Reply:
i thought to pass ctx timeout through struct once better than pass it though args for each funcs

Reply:
why do you need to pass cancel here?

Reply:
moved it to service layes